utils/gear-replay-cli/src/parse.rs

```diff
@@ -21,6 +21,23 @@
 use crate::{Block, BlockHashOrNumber, NumberFor};
 use hex::FromHexError;
 
+pub(crate) fn hash(block_hash: &str) -> Result<String, String> {
+    let (block_hash, offset) = if let Some(block_hash) = block_hash.strip_prefix("0x") {
+        (block_hash, 2)
+    } else {
+        (block_hash, 0)
+    };
+
+    if let Some(pos) = block_hash.chars().position(|c| !c.is_ascii_hexdigit()) {
```

Comment:
perhaps `match` is more appropriate for both cases in this func

Reply:
A matter of preference, I guess. Both versions seem to produce identical code when compiled.